quicksort/Cpp/quick_sort.cpp

```diff
@@ -0,0 +1,42 @@
+
+#include <iostream>
+
+using std::cout;
+
+void quick_sort(int a[], int left, int right) {
+
+      int i = left, j = right;
+      int pivot = a[right];
+
+      while (i <= j) {
+        while (a[i] < pivot)
+            i++; //carry on, this part already sorted
+        while (a[j] > pivot)
+            j--; //carry on, this part already sorted
+        if (i <= j) { // a[i]>=pivot and a[j]<=pivot
+            //swap a[i] with a[j]
+            int tmp = a[i];
+            a[i] = a[j];
+            a[j] = tmp;
+            i++;
+            j--;
+        }
+      };
+
+      if (left < j)
+        quick_sort(a, left, j);
+      if (i < right)
+        quick_sort(a, i, right);
+
```

Comment:
Redundant blank line at the end of a code block should be deleted.  [whitespace/blank_line] [3]

*Origin: CPPLintBear, Section: `all.cpplint`.*